### DISCUSSION TITLE
Getting legacy-install-failure when trying to install insiders

### DISCUSSION THREAD

**Discussion by zerenxyz** [squidfunk/mkdocs-material#4623] [Q&A] [answered]
I have insiders installed on a project and it works fine, however when installing it on a new project I get this error after running the pip install command from the docs

![image](https://user-images.githubusercontent.com/52886201/201143657-e6d580de-f999-4fc5-97fc-4843dcadfed9.png)

Running the same install and upgrade command for insiders on the previous project works just fine, so im lost

**Comment by squidfunk** [comment, marked as answer]
It looks like you need to install `libxml2`, which is necessary for `lxml`. It should be pre-installed on most systems. I'd just do what it says, i.e., execute `xcode-select --install`, which you need anyway to develop on a Mac. However, I'm not sure why it works on the same machine on another project.
